pkg/antctl/raw/proxy/command.go

```diff
@@ -78,7 +79,7 @@ func (o *proxyOptions) validateAndComplete() error {
 	}
 
 	if o.staticDir != "" {
-		fileInfo, err := os.Stat(o.staticDir)
+		fileInfo, err := osStat(o.staticDir)
 		if err != nil {
 			klog.Warningf("Failed to stat static file directory %s: %v", o.staticDir, err)
```

Comment:
we should update the legacy logs to use structured logging since we are editing this file